pkgs/development/interpreters/kamilalisp/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, jre, lib, makeWrapper }:
```

One per line, for better diffs

```suggestion
{ lib
, stdenv
, fetchurl
, jre
, makeWrapper
}:
```